apps/settings/templates/settings/admin/overview.php

```diff
@@ -30,7 +30,12 @@
 <div id="security-warning" class="section">
 	<div class="security-warning__heading">
 		<h2><?php p($l->t('Security & setup warnings'));?></h2>
-		<a target="_blank" rel="noreferrer" class="icon-info" title="<?php p($l->t('Open documentation'));?>" href="<?php p(link_to_docs('admin-warnings')); ?>"></a>
+		<a 	target="_blank"
+			rel="noreferrer"
+			class="icon-info"
+			title="<?php p($l->t('Open documentation'));?>"
+			href="<?php p(link_to_docs('admin-warnings')); ?>"
+			aria-label="<?php p($l->t('Open Documentation')); ?>"></a>
```

Comment:
```suggestion
			aria-label="<?php p($l->t('Open documentation')); ?>"></a>
```
nit: To keep casing consistent